### DISCUSSION TITLE
React does not recognize the isLazy prop on a DOM element

### DISCUSSION THREAD

**Discussion by Ali-Hussein-dev** [chakra-ui/chakra-ui#5027] [Q&A] [answered]
Warning from React "React does not recognize the `isLazy` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `islazy` instead. If you accidentally passed it from a parent component, remove it from the DOM element." 
When passing `islazy` prop with lowercase though, to the `Tabs` component React is still complaining, does someone know how to get rid of this nasty warning?

**Comment by primos63** [comment]
Can you provide an example of what you are doing? I tried the example from the [docs](https://chakra-ui.com/docs/disclosure/tabs#lazily-mounting-tab-panels) and didn't see any issue with `isLazy` on `Tabs`.

`Tab` itself does not accept `isLazy`. Placing `isLazy` on `Tab` would give you a warning.

**Comment by Ali-Hussein-dev** [comment]
```javascript
<Tabs
  colorScheme="sky"
  isFitted
  islazy
  variant={!isMobile ? 'enclosed-colored' : 'line'}
  className="relative"
>
// ...
</Tabs>
```

**Comment by primos63** [comment]
I think you mean to have `isLazy` rather than `islazy` as the latter is not a prop for `Tabs`. I took your implementation with minor modifications and I still get no error.
My changes: ` <Tabs colorScheme="blue" isFitted isLazy variant='enclosed-colored'>`

In your original post you said you were passing `isLazy` to `Tab` did you mean `Tabs` or do you have `isLazy` on another element within `Tabs`? That is the only way I'm able to get the same warning.

**Reply by Ali-Hussein-dev** [reply]
yes, I mean `Tabs`. 
I tried both `isLazy` & `islazy` but still getting the same error
I am using React and React-dom version: 17.0.2

**Comment by anubra266** [comment, marked as answer]
you're probably using `islazy` on another instance of `Tabs`. please check all instances in your application. capital `L`
